src/components/html-injection/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const widgetName = "html-injection";
```

Comment:
`export const widgetName = "html-injection";` I would also rename it to `custom-html-code` to match display name.